docker-compose.yaml

```
@@ -3,6 +3,9 @@ services:
         image: redis:alpine
         command: [ redis-server, --maxmemory 128mb, --maxmemory-policy volatile-lru, --save "" ]
 
+    rabbitmq:
+        image: rabbitmq:alpine
```

```suggestion
        image: rabbitmq:alpine
        volumes:
            - rabbitmq_data:/var/lib/rabbitmq/
```
I think we should use persistence as the default.

---

Done (but I used `pimcore-rabbitmq` instead to stay consistent). And it also works now with the latest docker prod image :+1: